dev-qt/qtsql/qtsql-4.8.7-r1.ebuild

```diff
@@ -18,6 +18,7 @@ REQUIRED_USE="
 
 DEPEND="
 	~dev-qt/qtcore-${PV}[aqua=,debug=,qt3support=,${MULTILIB_USEDEP}]
+	net-libs/libnsl:=[${MULTILIB_USEDEP}]
```

Comment:
are you sure qtsql directly depend on this?

Reply:
```
/var/tmp/portage/dev-qt/qtsql-4.8.7-r1/work/qt-everywhere-opensource-src-4.8.7/mkspecs/common/linux.conf
29:QMAKE_LIBS_NIS        = -lnsl
```

Reply:
Though I have no idea how QMAKE_LIBS_NIS ends up being linked into qtsql in the end. `./configure` has a `-no-nis` flag. Maybe that could be used to prevent this library from being linked in. But since I do not know about the consequences, I'd prefer if someone more experienced with dev-qt/qtsql could have a look at it first.

I also found this:
```
# mysql_config --libs
-L/usr/lib64  -lmariadb -lpthread -lnsl -lz -ldl -lm -lssl -lcrypto
```

So apparently dev-db/mariadb-10.2.11 links against net-libs/libnsl, which then propagates to dev-qt/qtsql-4.8.7-r1.

Reply:
> 29:QMAKE_LIBS_NIS        = -lnsl

Well, but this has nothing to do with qtsql... nis is handled in qtgui, and we're passing `-no-nis` elsewhere, although it looks like qtsql is missing that flag.

> So apparently dev-db/mariadb-10.2.11 links against net-libs/libnsl, which then propagates to dev-qt/qtsql-4.8.7-r1.

Then mariadb is missing the dependency.

Reply:
I reported this as https://bugs.gentoo.org/643038 and pushed 6d51438290ca97f07c9405c9ba42230392350e99, which does not include the (indirect) dependency anymore.